src/expmv_fun.jl

```diff
@@ -1,6 +1,6 @@
 export expmv
 
-function expmv(t, A, b; M = [], prec = "double", shift = false, full_term = false, prnt = false)
+function expmv(t, A, b; M::Array{Float64, 2} = Array(Float64, 0, 0), prec = "double", shift = false, full_term = false, prnt = false)
```

Comment:
Why not make `M` an optional argument instead so that way it dispatches. Then you can use `M=nothing` vs `M` a vector, and it will compile away the extranious branches when `M==nothing`.